pagopa/io-app
# [IOCOM-1370, IOCOM-1371] Selectable service name in message details (both standard and SEND), new DS

## Short description
This PR makes the service name selectable on the message details' screen, with the new DS on both standard and SEND messages (selection leads to service details' screen).
|Standard message|SEND message|
|-|-|
|![Simulator Screenshot - iPhone 15 - 2024-05-09 at 15 26 25](https://github.com/pagopa/io-app/assets/5150343/1e955740-252b-4fb2-92f8-0f6586550458)|![Simulator Screenshot - iPhone 15 - 2024-05-09 at 15 26 34](https://github.com/pagopa/io-app/assets/5150343/ba799363-3d85-4606-92fe-0814bbde7f38)|

## List of changes proposed in this pull request
- `OrganizationHeader` implements the `onPress` event on the service name, leading to the service details' screen
- `MessageDetailsHeader` adds the `serviceId` parameter to `OrganizationHeader` component

## How to test
Using the io-dev-api-server, check that both standard and SEND messages have a selectable service name that leads to the service details' screen
